portfolio/src/main/java/com/google/sps/servlets/PortfolioServlet.java

```diff
@@ -11,7 +14,23 @@
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse 
     response) throws IOException{
-           response.setContentType("text/html;");
-           response.getWriter().print("Scroll down to learn more!");
+           /*response.setContentType("text/html;");
+           response.getWriter().print("Scroll down to learn more!");*/
+           
+           //Array that contains hard-coded strings 
+           String facts[] = {"I am from Bulawayo in Zimbabwe!"," I am the last born child of 13 children!",
```

Comment:
Usually a space is after a comma when you initialize a collection

e.g.  {"I am from Bulawayo in Zimbabwe!", " I am the last born child of 13 children!", " My name is the name of a camera!" };